public/index.php

```diff
@@ -90,42 +135,17 @@
     for ($i = 0; $i < count($messageMedia); $i++){
       $mediaId = substr($messageMedia[$i], strpos($messageMedia[$i], "media/") + 6);
       $ext = substr($mediaId, strpos($mediaId, "."));
-      $response = $messagingClient->getMedia($BW_ACCOUNT_ID, $mediaId);
+      $mediaResponse = $messagingClient->getMedia($BW_ACCOUNT_ID, $mediaId);
       $file = fopen("media_file".$i.$ext, "wb") or die("Unable to open file");
-      fwrite($file, $response->getResult());
+      fwrite($file, $mediaResponse->getResult());
       fclose($file);
     }
   } else {
-    $myfile = fopen("outbound_message.txt", "w") or die("Unable to open file!");
-    $txt = "Type: ".$type."\nDescription: ".$description."\nText: ".$messageText;
+    $myfile = fopen("inbound_message.txt", "w") or die("Unable to open file!");
+    $txt = "Message type does not match endpoint. This endpoint is used for inbound messages only.\nOutbound message callbacks should be sent to /callbacks/outbound/messaging.";
     fwrite($myfile, $txt);
     fclose($myfile);
   }
 
   return $response->withStatus(200);
 });
-
-$app->post('/mediaManagement', function (Request $request, Response $response) {
-  global $messagingClient, $BW_ACCOUNT_ID;
-
-  $mediaName = "media_to_upload.jpg";
-  $data = $request->getBody();
-
-  // save image locally
-  $myfile = fopen($mediaName, "w") or die("Unable to open file!");
-  fwrite($myfile, $data);
-  fclose($myfile);
-
-  // upload image
-  $filename = $mediaName;
-  $file = fopen($filename, "rb") or die("Unable to open file");
-  $contents = fread($file, filesize($filename));
-  $messagingClient->uploadMedia($BW_ACCOUNT_ID, "bandwidth_sample_app.jpg", strlen($contents), $contents);
-  fclose($file);
-
-  // delete local image file
-  unlink($mediaName);
-  return $response->withStatus(200);
-});
-
-$app->run();
```

Comment:
Did you accidentally nuke `$app->run();`?

Reply:
😬